src/javascript/binary/websocket_pages/user/ico_subscribe.js

```diff
@@ -109,6 +110,36 @@ const ICOSubscribe = (() => {
         $total.html(formatMoney(currency, total));
     };
 
+    const ICOcountDown = () => {
+        const timer = $('.timer');
+        const days = timer.find('.time .days');
+        const hours = timer.find('.time .hours');
+        const minutes = timer.find('.time .minutes');
+        const seconds = timer.find('.time .seconds');
+        const timerID = window.setInterval(() => {
+            const start_time = 1510704000; 
+            const current_time = window.time.unix(); 
+            const time_left = start_time - current_time; 
+            if(time_left >= 0) {
+                timer.removeClass('invisible'); // Make the timer visible.
```

Comment:
can we make it visible only after all the text has been added? also you can use `.setVisibility(1)` instead of `.removeClass('invisible')`

Reply:
Sure @negar-binary